src/server/services/about/views/service.test.js

```diff
@@ -0,0 +1,214 @@
+import { nunjucksTestEnv } from '~/test-helpers/component-helpers.js'
+// eslint-disable-next-line n/no-unpublished-import
+import { toMatchFile } from 'jest-file-snapshot'
+
+expect.extend({ toMatchFile })
```

Comment:
I wonder should this sit in https://github.com/DEFRA/cdp-portal-frontend/blob/main/.jest/setup.js so all tests can use it?

Reply:
that's great!